src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -443,7 +450,16 @@ private static CalibratedModelParametersBase Create(IHostEnvironment env, ModelL
             var ver2 = GetVersionInfoBulk();
             var ver = ctx.Header.ModelSignature == ver2.ModelSignature ? ver2 : ver1;
             ctx.CheckAtModel(ver);
-            return new ValueMapperCalibratedModelParameters<TSubModel, TCalibrator>(env, ctx);
+
```

Comment:
I guess since they are identical model formats, my thought is that the only difference is the string. So it probably isn't worth it to try to test the `BCALPRED` path. You wouldn't be able to make one of these models - so you would have to somehow find an old one that was made this way. It probably isn't worth it.